src/Schema/Events/Tap.ts

```diff
@@ -204,7 +205,7 @@ export interface TappedEntityGroup {
   destination_screen_owner_type: ScreenOwnerType
   destination_screen_owner_id?: string
   destination_screen_owner_slug?: string
-  featured?: boolean
+  curation_boost?: boolean
```

Comment:
Does this field apply to all entity types?

Reply:
It could! But currently we only see this for artistSeries and artwork groups.